<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="labMap">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>557</width>
     <height>571</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="ckCanMove">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>510</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>可移动</string>
   </property>
  </widget>
  <widget class="DeviceButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DeviceButton" name="btn2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DeviceButton" name="btn3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>101</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnCircle">
      <property name="text">
       <string>圆形</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPolice">
      <property name="text">
       <string>警察</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnBubble">
      <property name="text">
       <string>气泡</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnBubble2">
      <property name="text">
       <string>气泡2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnMsg">
      <property name="text">
       <string>消息</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnMsg2">
      <property name="text">
       <string>消息2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnGreen">
      <property name="text">
       <string>绿色</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnBlue">
      <property name="text">
       <string>蓝色</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnRed">
      <property name="text">
       <string>红色</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnGray">
      <property name="text">
       <string>棕色</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnBlack">
      <property name="text">
       <string>黑色</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPurple">
      <property name="text">
       <string>紫色</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnYellow">
      <property name="text">
       <string>黄色</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DeviceButton</class>
   <extends>QToolButton</extends>
   <header location="global">D:\qt_project\control\DeviceButton\DeviceButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
